CMakeLists.txt

```diff
@@ -52,16 +52,16 @@ SET(VERSION_PATCH 0-git)
 include(GrVersion) #setup version info
 
 # Minimum dependency versions for central dependencies:
-set(GR_BOOST_MIN_VERSION "1.58")
-set(GR_CMAKE_MIN_VERSION "3.8.0")
-set(GR_MAKO_MIN_VERSION "0.4.2")
-set(GR_PYTHON_MIN_VERSION "3.6.5")
+set(GR_BOOST_MIN_VERSION "1.67")
+set(GR_CMAKE_MIN_VERSION "3.11.0")
+set(GR_MAKO_MIN_VERSION "1.0.7")
+set(GR_PYTHON_MIN_VERSION "3.7.3")
 set(GR_NUMPY_MIN_VERSION "1.13.0")
-set(GCC_MIN_VERSION "5.0.0")
-set(CLANG_MIN_VERSION "3.4.0")
+set(GCC_MIN_VERSION "8.3.0")
+set(CLANG_MIN_VERSION "7.0.1")
 set(APPLECLANG_MIN_VERSION "500")
 set(MSVC_MIN_VERSION "1900")
-set(VOLK_MIN_VERSION "2.1.0")
+set(VOLK_MIN_VERSION "2.3.0")
```

Comment:
Could we use VOLK 2.4.1 here?

Reply:
If we keep it down to 2.3.0, we can build GNU Radio on Ubuntu 20.10 without building VOLK from source, which would be a benefit. 

For GR3.10 development, we'll directly be upgrading to requiring 2.4.1.